### ISSUE TITLE
support for neovim winblend

### ISSUE THREAD

**Issue by luxus** [warpdotdev/Warp#1386]
hi
i have a bug with window blending in neovim with warp
https://github.com/LunarVim/LunarVim/issues/2641
as soon I disable 'winblend' it works
[winblend on neovim](https://neovim.io/doc/user/options.html#'winblend')

### To Reproduce

i think its easier to reproduce it with nvchad (easier to install)
the bug is related to [winblend](https://neovim.io/doc/user/options.html#'winblend') (a neovim function) and is not related to a specific config or plugin

* install neovim with brew
```sh
brew install neovim
```
* install Nvchad a neovim config
```sh
git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1 && nvim
```
* edit whichkey config
```sh
$EDITOR ~/.config/nvim/lua/plugins/configs/whichkey.lua
```
* goto line 41 and add change it to this:
```lua
 window = {
      border = "none", -- none/single/double/shadow
      winblend = 20,
 },
```
* open any bigger file (that fills the screen)
```sh
nvim ~/.config/nvim/lua/plugins/configs/whichkey.lua
```
* press `space` to open the which key menu
it should look like this:
<img width="406" alt="Screenshot 2022-05-26 at 03 58 42" src="https://user-images.githubusercontent.com/7449/170399724-7d6de95c-9a9f-4ceb-810f-b5a17dc97349.png">

in warp it looks like this:
<img width="467" alt="Screenshot 2022-05-26 at 03 55 26" src="https://user-images.githubusercontent.com/7449/170399379-cfe45f32-6768-4a07-b144-d54dc1282f18.png">

to close nvim as a non vim user enter `:q!`

### OS Version

12.4

### Shell Version

v0.2022.05.16.09.01.stable_01

**Comment by vorporeal** [comment]
Mind providing your lunarvim configuration and some repro steps?  I don't personally use neovim/lunarvim, so don't know what to enable in the config in order to get a pop-up with pseudo-transparency.

**Comment by luxus** [comment]
@vorporeal sure.. i'm happy to help

i think its easier to reproduce it with nvchad (easier to install)
the bug is related to [winblend](https://neovim.io/doc/user/options.html#'winblend') (a neovim function) and is not related to a specific config or plugin

* install neovim with brew
```sh
brew install neovim
```
* install Nvchad a neovim config
```sh
git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1 && nvim
```
* edit whichkey config
```sh
$EDITOR ~/.config/nvim/lua/plugins/configs/whichkey.lua
```
* goto line 41 and add change it to this:
```lua
 window = {
      border = "none", -- none/single/double/shadow
      winblend = 20,
 },
```
* open any bigger file (that fills the screen)
```sh
nvim ~/.config/nvim/lua/plugins/configs/whichkey.lua
```
* press `space` to open the which key menu
it should look like this:
<img width="406" alt="Screenshot 2022-05-26 at 03 58 42" src="https://user-images.githubusercontent.com/7449/170399724-7d6de95c-9a9f-4ceb-810f-b5a17dc97349.png">

in warp it looks like this:
<img width="467" alt="Screenshot 2022-05-26 at 03 55 26" src="https://user-images.githubusercontent.com/7449/170399379-cfe45f32-6768-4a07-b144-d54dc1282f18.png">


to close nvim as a non vim user enter `:q!`

**Comment by vorporeal** [comment]
Thanks for the repro steps!  I've figured out what's going on here; discussing with the team to see how we want to fix it.  Hopefully will have a fix out for you soon. :)

**Comment by vorporeal** [comment]
Submitted a fix; should go live with next week's release!  I'll keep this open until you can confirm that it actually fixes the issue for you. :)

**Comment by luxus** [comment]
finally i can open warp again in ventura..
sadly the bug is not fixed @vorporeal

**Comment by luxus** [comment]
any update @vorporeal ? i tested it today and it is still glitching :(

**Comment by axynos** [comment]
Popping in to say that on `v0.2022.08.08.09.21.stable_00` it's still glitching when using a non 0/100 value for pumblend. (such as `pumblend=15`).

**Comment by luxus** [comment]
its fixed :D v0.2022.11.14.14.55.stable_01
